opm/autodiff/BlackoilWellModel_impl.hpp

```diff
@@ -289,8 +289,16 @@ namespace Opm {
     }
 
 
-
-
+    template<typename TypeTag>
+    void
+    BlackoilWellModel<TypeTag>::
+    addWellContributions(Mat& mat) const
```

Comment:
It looks to me like this method never gets called; instead the individual wells' `addWellContributions()` get called inside `StandardWell<TypeTag>::assembleWellEq()`. So this method should be removed. Also, I think that the decision to call (or not) addWellContributions() could be made on this level, in the assembleWellEq() method of this class, rather than in the StandardWell class. Wouldn't that be better?

Reply:
Yep that was a previous design. Gone now.